build_in_docker/file_cavity_detection.py

```diff
@@ -351,12 +351,7 @@ def file_cavity_detection(
             print(f"Error when processing {filename}")
             return f"{filename},-3,-3\n"
 
-        command = [
-            "python3",
-            str(SCRIPTDIR / "../polytracker/dumptdag.py"),
-            str(dst_tdag),
-            str(tool.container_input_path(filename)),
-        ]
+        command = ["polytracker", "cavities", str(dst_tdag)]
```

Comment:
This is a slight change in behavior, right? Previously we only computed cavities for the processed file. If I don't misread, now we would get cavities for all files being processed by the command.

Reply:
All file descriptors tracked in the `.tdag` file, but yes. I think it shouldn't be a problem, unless the tool introduces taint sources from a file not specified by `filename`. That doesn't happen with the tools we currently support in the `file_cavity_detection.py` script, right? Correct me if I'm wrong though! 

One thing I could do is save us a shell spawn, detect cavities using API and filter out results for `filename`.

Reply:
I think your last point would be a great addition!

Reply:
Turns out the implementing timeouts for threads is more work than it's worth. I take  my suggestion back :sweat_smile: